src/h5web/App.tsx

```diff
@@ -44,7 +44,7 @@ function App() {
         />
         <ErrorBoundary
           resetKeys={[selectedPath, isInspecting]}
-          FallbackComponent={ErrorMessage}
+          FallbackComponent={ErrorFallback}
```

Comment:
Renaming this component to distinguish it from the domain slider's `ErrorMessage` component. `ErrorFallback` (passed to `ErrorBoundary`) is consistent with `LoadingFallback` (passed to `Suspense`).